SU-SWS/stanford_paragraph_types
# EARTH-83: Removed more default lables from showing up.

# READY FOR REVIEW

# Summary
- Removes the default labels from the top of each paragraph. 

# Needed By (Date)
- June 2nd

# Urgency
- Not very

# Steps to Test

1. Refresh local environment (blt local:refresh)
2. Create a complex page with a few of the paragraph types and see the default labels on them
3. Check out this branch
4. Import all features (drush features-import-all)
5. Clear caches
6. View page with paragraphs and the default labels should be gone.

# Affected Projects or Products
- EARTH-83

# Associated Issues and/or People
- EARTH-83

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
